statics/js/components/topology.js

```diff
@@ -238,6 +249,7 @@ var TopologyComponent = {
 
   beforeDestroy: function() {
     this.$store.commit('unselected');
```

Comment:
could be nice to rename "unselected" to "nodeUnselected"

Reply:
done